go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.8
 
 require (
 	github.com/docker/docker v26.1.5+incompatible
-	github.com/elastic/elastic-agent-libs v0.17.4-0.20241126154321-6ed75416832d
+	github.com/elastic/elastic-agent-libs v0.17.4
```

Comment:
Is this going to remove the PDH stuff you did?  I think we might need a new release of elastic-agent-libs.